oregon-a/basket-dev/web-deploy.yaml

```diff
@@ -48,366 +37,226 @@ spec:
               key: database-url
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: DEBUG
-          valueFrom:
-            secretKeyRef:
-              key: debug
-              name: basket-dev-v107-env
-        - name: DEIS_APP
-          valueFrom:
-            secretKeyRef:
-              key: deis-app
-              name: basket-dev-v107-env
-        - name: DEIS_DOMAIN
-          valueFrom:
-            secretKeyRef:
-              key: deis-domain
-              name: basket-dev-v107-env
+          value: "False"
         - name: DONATE_ACCESS_KEY_ID
           valueFrom:
             secretKeyRef:
               key: donate-access-key-id
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: DONATE_CONTACT_RECORD_TYPE
-          valueFrom:
-            secretKeyRef:
-              key: donate-contact-record-type
-              name: basket-dev-v107-env
+          value: 012U00000001rnrIAA
         - name: DONATE_NOTIFY_EMAIL
           valueFrom:
             secretKeyRef:
               key: donate-notify-email
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: DONATE_OPP_RECORD_TYPE
-          valueFrom:
-            secretKeyRef:
-              key: donate-opp-record-type
-              name: basket-dev-v107-env
+          value: 0120B000000t6NVQAY
         - name: DONATE_QUEUE_IGNORE_MODE
-          valueFrom:
-            secretKeyRef:
-              key: donate-queue-ignore-mode
-              name: basket-dev-v107-env
+          value: "True"
         - name: DONATE_QUEUE_REGION
-          valueFrom:
-            secretKeyRef:
-              key: donate-queue-region
-              name: basket-dev-v107-env
+          value: us-east-1
         - name: DONATE_QUEUE_URL
           valueFrom:
             secretKeyRef:
               key: donate-queue-url
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: DONATE_SECRET_ACCESS_KEY
           valueFrom:
             secretKeyRef:
               key: donate-secret-access-key
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: EMAIL_HOST
           valueFrom:
             secretKeyRef:
               key: email-host
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: EMAIL_HOST_PASSWORD
           valueFrom:
             secretKeyRef:
               key: email-host-password
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: EMAIL_HOST_USER
           valueFrom:
             secretKeyRef:
               key: email-host-user
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: EMAIL_PORT
-          valueFrom:
-            secretKeyRef:
-              key: email-port
-              name: basket-dev-v107-env
+          value: 587
```

Comment:
I believe leaving the quotes off of this number will result in a type error, since an env var need to be a string